src/app/extensions/wishlists/facades/wishlists.facade.ts

```diff
@@ -69,8 +69,8 @@ export class WishlistsFacade {
     this.store.dispatch(addProductToNewWishlist({ title, sku }));
   }
 
-  addProductToWishlist(wishlistId: string, sku: string, quantity?: number): void {
-    this.store.dispatch(addProductToWishlist({ wishlistId, sku, quantity }));
+  addProductToWishlist(wishlistId: string, sku: string): void {
```

Comment:
Why is the quantity attribute removed, although the property could be added as a query param to the addProductToWishlist REST call?